patches/0056-ipq40xx-change-EC420-G1-patch_mac-method.patch

```diff
@@ -0,0 +1,38 @@
+From 73bb7009f980da3e898b590be06328703235c8ee Mon Sep 17 00:00:00 2001
+From: Arthur Su <arthur.su@tp-link.com>
+Date: Fri, 16 Jul 2021 09:43:25 +0000
+Subject: [PATCH] ipq40xx: change EC420-G1 patch_mac method
+
+deprecate get_tip command, because manufacturing data aren't binrary
+raw.
+
+Signed-off-by: Arthur Su <arthur.su@tp-link.com>
+---
+ .../base-files/etc/hotplug.d/firmware/11-ath10k-caldata       | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata b/target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata
```

Comment:
Do we need a change similar to this in target/linux/ipq807x/base-files/etc/hotplug.d/firmware/10-ath11k-caldata to setup MAC addresses for for the ex227 and ex447?

Reply:
I have added another patch for ex227 and ex447 , and the 10-ath11k-caldata can setup wlan0 and wlan1 mac address for tplink devices.